apps/dashboard/src/pages/usecase-select-page.tsx

```diff
@@ -0,0 +1,141 @@
+import { UsecaseSelectOnboarding } from '../components/auth/usecase-selector';
+import { AuthCard } from '../components/auth/auth-card';
+import { useState } from 'react';
+import { motion, AnimatePresence } from 'framer-motion';
+import { Button } from '../components/primitives/button';
+import { ROUTES } from '../utils/routes';
+import { useNavigate } from 'react-router-dom';
+import { OnboardingArrowLeft } from '../components/icons/onboarding-arrow-left';
+import { updateClerkOrgMetadata } from '../api/organization';
+import { ChannelTypeEnum } from '@novu/shared';
+import { RiLoader2Line } from 'react-icons/ri';
+import { PageMeta } from '../components/page-meta';
+import { useTelemetry } from '../hooks';
+import { TelemetryEvent } from '../utils/telemetry';
+import { channelOptions } from '../components/auth/usecases-list.utils';
+
+export function UsecaseSelectPage() {
+  const navigate = useNavigate();
+  const track = useTelemetry();
+  const [loading, setIsLoading] = useState(false);
+  const [selectedUseCases, setSelectedUseCases] = useState<ChannelTypeEnum[]>([]);
+  const [hoveredUseCase, setHoveredUseCase] = useState<ChannelTypeEnum | null>(null);
+
+  const displayedUseCase = hoveredUseCase || (selectedUseCases.length > 0 ? selectedUseCases[0] : null);
+
+  async function handleContinue() {
+    setIsLoading(true);
+
+    try {
+      await updateClerkOrgMetadata({
+        useCases: selectedUseCases,
+      });
+
+      track(TelemetryEvent.USE_CASE_SELECTED, {
+        useCases: selectedUseCases,
+      });
+      navigate(ROUTES.WORKFLOWS);
+    } catch (error) {
+      console.error(error);
+    } finally {
+      setIsLoading(false);
+    }
+  }
+
+  function handleSkip() {
+    track(TelemetryEvent.USE_CASE_SKIPPED);
+
+    navigate(ROUTES.WORKFLOWS);
+  }
+
+  function handleSelectUseCase(useCase: ChannelTypeEnum) {
+    setSelectedUseCases((prev) =>
+      prev.includes(useCase) ? prev.filter((useCase) => useCase !== useCase) : [...prev, useCase]
+    );
+  }
+
+  return (
+    <>
+      <PageMeta title="Customize you experience" />
+
+      <AuthCard>
+        <div className="flex w-[480px] justify-center px-0">
+          <div className="flex max-w-[480px] flex-col items-center justify-center gap-8 p-[60px]">
+            <UsecaseSelectOnboarding
+              channelOptions={channelOptions}
+              selectedUseCases={selectedUseCases}
+              onHover={(id) => setHoveredUseCase(id)}
+              onClick={(id) => handleSelectUseCase(id)}
+            />
+
+            <div className="flex w-full flex-col items-center gap-3">
+              <Button disabled={selectedUseCases.length === 0 || loading} className="w-full" onClick={handleContinue}>
```

Comment:
Without using the form the keyboard enter won't work.

Reply:
Made the whole form accessible with tab navigation aswell 💪